tests/sentry/incidents/action_handlers/test_email.py

```diff
@@ -168,13 +197,29 @@ def test_team_alert_disabled(self):
             user_id=self.user.id,
             project=self.project,
         )
+        NotificationSettingProvider.objects.create_or_update(
+            user_id=self.user.id,
+            scope_type=NotificationScopeEnum.PROJECT.value,
```

Comment:
@snigdhas in reality, we don't ever create `NotificationSettingProvider` with the project scope. Do we need this?